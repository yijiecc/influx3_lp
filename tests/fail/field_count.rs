use influx3_lp::Influx3Lp;

#[derive(Influx3Lp)]
#[influx3_lp(table_name = "home")]
struct SensorData {
}    

