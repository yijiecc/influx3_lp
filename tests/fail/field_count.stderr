error: proc-macro derive panicked
 --> tests/fail/field_count.rs:3:10
  |
3 | #[derive(Influx3Lp)]
  |          ^^^^^^^^^
  |
  = help: message: SensorData should have at least one field

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/fail/field_count.rs:6:2
  |
6 | }
  |  ^ consider adding a `main` function to `$DIR/tests/fail/field_count.rs`
